src/app/productDetail/templates/productDetail.images.html

```diff
@@ -0,0 +1,13 @@
+<div cg-busy="carouselLoading" ng-if="product.xp && product.xp.additionalImages && product.xp.additionalImages.length">
```

Comment:
@amrarick26 You should do your ngRepeat over a new array variable that combines the default image (product.xp.image) with the additionalImages array. Make the default image the first item in the array so that it is active by default.

This new array is also what should be passed into the product images modal to get the same result there.